espanol/capitan/trueno.md

```diff
@@ -2,4 +2,10 @@ El Capitán Trueno estaba volviendo en ese momento de su campaña en las tierras
 
 En su camino de regreso se encontro con una bruja. 
 
-[Una bruja ofrece ayuda](../bruja/bruja.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.